hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/TestScannersWithFilters.java

```diff
@@ -1,5 +1,4 @@
-/*
- *
+/**
```

Comment:
this is a file header and not a javadoc, shouldn't we be using `/*` and not `/**`?

Reply:
Fixed